src/protobot_bringup/launch/protobot_core.launch

```diff
@@ -3,4 +3,5 @@
 	<include file="$(find protobot_localization)/launch/protobot_localization.launch"/>
 	<include file="$(find protobot_mapping)/launch/protobot_mapping.launch"/>
 	<include file="$(find protobot_control)/launch/protobot_control.launch"/>
+	<include file="$(find protobot_hardware)/launch/protobot_hardware.launch"/>
```

Comment:
no

Reply:
protobot_hardware is not a core item since it is substituted by protobot_gazebo, launch protobot_onboard for hardware